pyaedt/application/Design.py

```diff
@@ -1754,22 +1770,22 @@ def change_material_override(self, material_override=True):
     @aedt_exception_handler
     def change_validation_settings(self, entity_check_level="Strict", ignore_unclassified=False,
                                    skip_intersections=False):
-        """change the validation design settings
+        """Update the validation design settings.
 
         Parameters
         ----------
-        entity_check_level : str
-            Entity Level, default "Strict"
-        ignore_unclassified : bool
-            Ignore unclassified elements (Default value = False)
-        skip_intersections : bool
-            Skip Intersections (Default value = False)
+        entity_check_level : str, optional
+            Entity Level. The default is ``"Strict"``.
+        ignore_unclassified : bool, optional
+            Whether to ignore unclassified elements. The default is ``False``.
+        skip_intersections : bool, optional
+            Whether to skip intersections. The default is ``False``.
 
         Returns
         -------
-
-            bool
-
+        bool
+            ``True`` when successful, ``False`` when failed.
+            
```

Comment:
```suggestion

```

I'm fine with having an additional line at the end of the docstring (and it helps to break the docs from the source).  Just let's make sure there isn't extra whitespace.  GitHub adds it automatically, so check for that.